vkconfig/main_doc.cpp

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2020-2022 Valve Corporation
```

Comment:
For a new file, why does the copyright go back to 2020?

Reply:
Because the file is a port of another file... I'll change it.